README.md

```
@@ -1,36 +1,102 @@
+## Running Tests
+
+Tests include:
+
+- REST endpoint testing (GET, POST, DELETE)
+- Spring Security access control
+- Input validation
+- Authenticated vs anonymous requests
+
+## Web Interface
+
+Navigate to http://localhost:8080 to:
```

markdownlint-cli2 (0.18.1)

Bare URL used

(MD034, no-bare-urls)